### DISCUSSION TITLE
Trying to design simpler inner join

### DISCUSSION THREAD

**Discussion by Dispersia** [SeaQL/sea-orm#728] [Ideas]
Hello, I'm trying to do a similar concept to find_also_related, but for inner joins instead of outer (which can then return a tuple of (element, element) instead of (element, Option<element>). However, I don't understand where the Option part of that query is actually coming from (from select_also). I'm still not great at following traits like this so was just hoping to have some guidance on that, or if this would be a good idea to have through a PR.

Current implementation is just
```
pub fn find_also_inner_related<R>(self, r: R) -> SelectTwo<E, R>
where
    R: EntityTrait,
    E: Related<R>,
{
    self.join_join(JoinType::InnerJoin, E::to(), E::via())
       .select_also(r)
}
```

**Comment by billy1624** [comment]
Hey @Dispersia, welcome!

The code snippet that control returning `(element, Option)` was
- https://github.com/SeaQL/sea-orm/blob/a517e9db1aa8447300c25c3cf7fcf06724d0b698/src/executor/select.rs#L97-L110

I think it's good that we can a public API specifically for mandatory relation. A PR would be welcomed!
Feel free to let us know if you need any help :P
